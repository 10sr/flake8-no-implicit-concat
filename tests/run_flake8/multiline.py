a = ["aaa",
     "bbb"
     "ccc"]

b = ("lll"
     "mmm",
     "nnn")

def f():
    a = ["aaa",
         "bbb"
         "ccc"]
    return a

print("abc"
      "def",
      "ghi")
