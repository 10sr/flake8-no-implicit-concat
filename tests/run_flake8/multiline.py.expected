./multiline.py:2:11: NIC002 Implicitly concatenated str literals over multiple lines
./multiline.py:5:11: NIC002 Implicitly concatenated str literals over multiple lines
./multiline.py:11:15: NIC002 Implicitly concatenated str literals over multiple lines
./multiline.py:15:12: NIC002 Implicitly concatenated str literals over multiple lines
